src/coreclr/tools/ILVerification/ILImporter.Verify.cs

```diff
@@ -2647,6 +2648,18 @@ void CheckPendingPrefix(Prefix mask)
             Check((mask & Prefix.Constrained) == 0, VerifierError.Constrained);
         }
 
+        static bool HasIsExternalInit(MethodSignature signature)
+        {
+            if (signature.HasEmbeddedSignatureData && signature.GetEmbeddedSignatureData() != null)
+                foreach (var data in signature.GetEmbeddedSignatureData())
+                {
+                    if (data.type is MetadataType mdType && mdType.Name == "IsExternalInit")
```

Comment:
Does this need to check the namespace as well?

Reply:
Sure, I’ll double check. I dumped this object to find difference with and without modreq. Ignored the fields which weren’t serializing; I guess namespace was one of them.